package.json

```diff
@@ -17,6 +17,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.34.2",
+    "react-hook-form-resolvers": "^0.0.1",
```

Comment:
Zerknij sobie czy to jest na pewno to co chciałeś dodać do projektu bo dziwnie ta paczka wygląda